galleries/plot_types/basic/scatter_plot.py

```diff
@@ -11,12 +11,14 @@
 plt.style.use('_mpl-gallery')
 
 # make the data
+rng = np.random.default_rng(seed=None)
```

Comment:
```suggestion
rng = np.random.default_rng(seed=19680801)
```

We use 19680801 as the seed anyplace we have to put in a seed.

Reply:
This is not true for the "Plot types" examples. They all use 3, likely because they were taken from the cheat sheets. I'm in (weak) favour of keeping that (a) for consistency and (b) because 19680801 is more noisy.

Reply:
@qqwqqw689 This needs a seed.  I'd use "3" in this case as well.  

I'm not clear why you have only changed one example.